ClassAttendance/src/ConsoleUI/ConsoleUI.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Linq.Async" Version="4.1.1" />
```

Comment:
Where do you use this package?

Reply:
IAsyncEnumerable doesn't have ToListAsync(), this  package adds it.